packages/documentation/src/stories/components/alert/web-component/post-alert.stories.ts

```diff
@@ -0,0 +1,142 @@
+import { Meta, StoryContext, StoryFn, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import { BADGE } from '../../../../../.storybook/constants';
+import { spreadArgs } from '../../../../utils';
+import './post-alert.styles.scss';
+
+const meta: Meta<HTMLPostAlertElement> = {
+  title: 'Components/Post Alert',
+  component: 'post-alert',
+  render: renderAlert,
+  decorators: [externalControl],
+  parameters: {
+    badges: [BADGE.NEEDS_REVISION],
+  },
+  args: {
+    dismissible: false,
+    dismissLabel: 'Dismiss',
+    fixed: false,
+    innerHTML: '<p>Contentus momentus vero siteos et accusam iretea et justo.</p>',
```

Comment:
I'm not sure. Is those p tag required?

Reply:
It is required for accessibility purposes.

Reply:
For me it's debatable in this case. As far as I know, using `<p>` is good, as you can navigate between `<p>` in a page with accessibility tools. In this precise case, there is only one and no heading, so I'm not sure if it's that useful. But it doesn' have negative impact I think.